docs/cache/develop/api-reference/collection-ttl.md

```diff
@@ -17,24 +17,28 @@ The CollectionTtl type is used when performing write operations on a collection.
 Sometimes, when you update a collection, you want to refresh the TTL. Other times you want to keep the TTL the same. The
 CollectionTtl parameter allows you to specify this behavior.
 
-The default behavior is that the collection TTL is modified whenever a write operation occurs.
+The default behavior is that the collection TTL is reset whenever a write operation occurs. You cannot provide a CollectionTTL object when performing a read operation like `dictionaryFetch` or `listLength`. 
 
 See [Expire Data with TTL](./../../learn/how-it-works/expire-data-with-ttl.md) for more information on how TTL works with Momento Cache.
 
+:::caution
+
+The CollectionTTL specifies the TTL for the entire collection, *not* for setting the TTL of individual elements within a collection. The elements within will not expire if the collection does not expire.
+
+:::
+
 ## Compatible data types
 
-The CollectionTTL object is compatible with the following data types *when performing write operations*:
+The CollectionTTL object is compatible with the following [collection data types](/cache/develop/basics/datatypes#collection-data-types-cdts) *when performing write operations*:
 
 * [Dictionary](../api-reference/dictionary-collections.md)
 * [List](../api-reference/list-collections.md)
 * [Set](../api-reference/set-collections.md)
 * [Sorted Set](../api-reference/sorted-set-collections.md)
 
-:::info
-
-You cannot provide a CollectionTTL object when performing a read operation like `dictionaryFetch` or `listLength`. 
+As the image below illustrates, collections are considered `items` that may contain `elements`. 
```

Comment:
added the graphic showing that collections == items, but mostly I tried to just refer to sorted sets as "collections" throughout